src/test/regress/expected/multi_sequence_default.out

```diff
@@ -127,44 +124,56 @@ SELECT * FROM seq_test_0_local_table ORDER BY 1, 2 LIMIT 5;
 ---------------------------------------------------------------------
  integer |     1 |       1 | 2147483647 |         1 | no      |     1
 
--- cannot change the type of a sequence used in a distributed table
--- even if metadata is not synced to workers
+-- cannot alter a sequence used in a distributed table
+-- since the metadata is synced to workers
 ALTER SEQUENCE seq_0 AS bigint;
-ERROR:  Altering a sequence used in a distributed table is currently not supported.
+ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0_local_table AS bigint;
-ERROR:  Altering a sequence used in a local table that is added to metadata is currently not supported.
--- we can change other things like increment
--- if metadata is not synced to workers
+ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0 INCREMENT BY 2;
+ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0_local_table INCREMENT BY 2;
+ERROR:  Altering a distributed sequence is currently not supported.
 \d seq_0
                   Sequence "sequence_default.seq_0"
   Type   | Start | Minimum |  Maximum   | Increment | Cycles? | Cache
 ---------------------------------------------------------------------
- integer |     1 |       1 | 2147483647 |         2 | no      |     1
+ integer |     1 |       1 | 2147483647 |         1 | no      |     1
 
 \d seq_0_local_table
             Sequence "sequence_default.seq_0_local_table"
   Type   | Start | Minimum |  Maximum   | Increment | Cycles? | Cache
 ---------------------------------------------------------------------
- integer |     1 |       1 | 2147483647 |         2 | no      |     1
+ integer |     1 |       1 | 2147483647 |         1 | no      |     1
 
--- check that we can add serial pseudo-type columns
--- when metadata is not yet synced to workers
+-- check that we cannot add serial pseudo-type columns
+-- since the metadata is synced to workers
 TRUNCATE seq_test_0;
```

Comment:
Well, these tests become useless now. Can we do something like:

BEGIN;
SELECT stop_metadata_sync_to_node();
....
other commands...
ROLLBACK;